lib/models/sagemaker-schedule.ts

```diff
@@ -0,0 +1,48 @@
+import * as cdk from 'aws-cdk-lib';
+import * as scheduler from 'aws-cdk-lib/aws-scheduler';
+import * as iam from 'aws-cdk-lib/aws-iam';
+import * as sagemaker from "aws-cdk-lib/aws-sagemaker";
+import { Construct } from 'constructs';
+
+export function createStartSchedule(scope: Construct, id: string, sagemakerEndpoint: sagemaker.CfnEndpoint, role: iam.Role, scheduleExpression: string) {
```

Comment:
To follow suit with how other CDK constructs do it, rather than put all the params here as positional args, I would build a "Props" type and use that:

```
export interface ScheduleProps {
    sagemakerEndpoint: sagemaker.CfnEndpoint;
    role: iam.Role;
    scheduleExpression: string;
}

export function createStartSchedule(scope: Construct, id: string, props: ScheduleProps) {
```